### PR TITLE
new package: focuswriter

### PR DESCRIPTION
**Pull request by Piraty**

### CODE REVIEW

File: srcpkgs/focuswriter/template [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+revision=1
+build_style=qmake
+hostmakedepends="pkg-config qt5-qmake qt5-tools-devel"
+makedepends="qt5-devel qt5-multimedia-devel hunspell-devel pulseaudio-devel"
+short_desc="Simple, distraction-free word processor with a hide-away interface"
+maintainer="Piraty <piraty1@inbox.ru>"
+license="GPL-3"
+homepage="https://gottcode.org/focuswriter/"
+distfiles="https://gottcode.org/focuswriter/download/?os=source&tip=0>focuswriter.tar.gz"
+distfiles="https://github.com/gottcode/${pkgname}/archive/v${version}.tar.gz"
```
Comment (lemmi) [inline srcpkgs/focuswriter/template]: Which of the `distfiles` is it?

File: srcpkgs/focuswriter/template [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Template file for 'focuswriter'
+pkgname=focuswriter
+version=1.6.6
```
Comment (lemmi) [inline srcpkgs/focuswriter/template]: 1.6.7 seems to be the current version.

Comment (Piraty) [reply srcpkgs/focuswriter/template]: yeah, they released it after initial contribution of this template

File: srcpkgs/focuswriter/template (hunk 1 above)
Comment (Piraty) [reply srcpkgs/focuswriter/template]: sorry I left the old one in, it's the one from the homepage.
I go for the github.
thanks for review

**Comment by lemmi** [comment]
Closes #7087.

**Pull request closed without merging**